common/lib/api-client/models.js

```diff
@@ -1,74 +1,71 @@
-function defineModels(jsonApi) {
-  jsonApi.define('move', {
-    reference: '',
-    status: '',
-    move_type: '',
-    additional_information: '',
-    updated_at: '',
-    time_due: '',
-    date: '',
-    person: {
-      jsonApi: 'hasOne',
-      type: 'people',
+module.exports = {
+  move: {
```

Comment:
maybe just preference but how come these are being set to empty strings arther than say `null`?

Reply:
This was just following the [devour docs](https://github.com/twg/devour#quick-start).